lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
Let me back up a bit. I think it's actually _possible_ to take care of the parallel behavior in the `package.py` with `self.make_jobs`. But I wouldn't do that in this PR. Bazel should be treated like a first-class build system, even if it's not very popular yet. In fact, I think SCons is only used in 2 packages at the moment. Your argument for moving all of the build system stuff into the `package.py` files is kind of like the argument to make compilers ordinary virtual dependencies. So you could possibly convince me that we should move all of this stuff to the `package.py` files. But if we do that, it should be for all build systems, not just Bazel. This PR introduces support for Bazel. Another PR could move build system configuration to the `package.py` files. But specifying everything in `build_environment.py` is a lot less duplicated code.